cnd/src/network.rs

```diff
@@ -1188,90 +1197,82 @@ impl libp2p::swarm::NetworkBehaviourEventProcess<comit_ln::BehaviourOutEvent> fo
                 let role = swap_params.role;
                 let create_swap_params = swap_params;
```

Comment:
I know you didn't touch this but I don't see the reason for it, the prefix 'create_' adds no extra information IMO.